lib/ut-coverage/test_v7_bplib.c

```diff
@@ -43,48 +52,106 @@ void test_bplib_deinit(void)
 void test_bplib_create_ram_storage(void)
 {
     /* bp_handle_t bplib_create_ram_storage(bplib_routetbl_t *rtbl, const bp_ipn_addr_t *storage_addr) */
+    bplib_routetbl_t *rtbl = NULL;
+    bp_ipn_addr_t storage_addr;
+    bp_handle_t reval = bplib_create_ram_storage(rtbl, &storage_addr);
+    UtAssert_UINT32_EQ(reval.hdl, 0);
```

Comment:
GCC 11 issues a warning here that `storage_addr` is used uninitialized - just need to initialize it before passing in.